velox/connectors/hive/storage_adapters/s3fs/RegisterS3FileSystem.cpp

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+#include "velox/connectors/hive/storage_adapters/s3fs/RegisterS3FileSystem.h" // @manual
```

Comment:
@majetideepak How about moving this line after https://github.com/facebookincubator/velox/pull/12217/files#diff-3bb861689e89d1ad4e2aa6178aea365723040a78e0ea8907c5f819d52f8aeceaR19? This way, we can include this file only when the S3 file system is enabled.

Reply:
@JkSelf this is a public header and has the definition of `CacheKeyFn`. So it is okay to be not included in the ifdef.